_pages/en/strategy-build-vs-buy.md

```diff
@@ -224,14 +314,14 @@ For details as to why the current approach to build vs buy decisions is out of a
 
 For further details regarding the increase of build decisions (based on the new definition of build, as defined in the Definitions section), reference Annex 3: Alignment.
 
-##### 3. Use Service Oriented Architecture (SOA)
+##### 3. Use Service Oriented Architecture (SOA)<!-- omit in toc -->
 
 Given the aforementioned principle, Buy software services -- not products, when seeking a vendor to build a given component, industry best practices[^11] will be adhered to by following a Service Oriented Architecture.
 This equips the department to be better able to align with the last principle outlined, _Minimize the size, cost, and scale_.
 By breaking up the monolithic solution into interchangeable sub-components, this increases the flexibility and scalability of the solutions, thereby improving the quality and reliability[^12] of digital services delivered to Canadians.
 By adhering to this approach enterprise software may have subsets of its functionality regularly updated to avoid requiring expensive protracted initiatives to revive expiring systems after decades of neglect due to the fear of complexity of the monolith[^13].
 
-##### 4. Publish as open source
+##### 4. Publish as open source<!-- omit in toc -->
```

Comment:
remove from Appendix A (diagnostic). This appendix should only list problem statements (the diagnosis) that is leading us to make policy decisions (the strategy).

Should add it in the guiding policy under Architecture and Legal (new recommended) sections.
And reference the ESDC OSS Framework

Reply:
These are also already established TBS Policy instruments, I'm not sure repeating it over is required :thinking:

Reply:
Removed from now but added the reference to the ESDC OSS Framework.